examples/apps/ai-collab/src/components/UserProfilePhoto.tsx

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InteractiveBrowserCredential } from "@azure/identity";
+import { Client } from "@microsoft/microsoft-graph-client";
+import { TokenCredentialAuthenticationProvider } from "@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials";
+import { Avatar, Badge } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+
+import type { UserPresence } from "@/app/tasks-list/presence";
+
+interface UserProfilePhotoProps {
+	onlineUsers: UserPresence;
+}
+
+const UserProfilePhoto: React.FC<UserProfilePhotoProps> = ({ onlineUsers }) => {
+	const AZURE_CLIENT_ID = process.env.AZURE_CLIENT_ID;
+	if (AZURE_CLIENT_ID === undefined) {
+		throw new Error("AZURE_CLIENT_ID environment variable is not set");
+	}
+
+	const AZURE_TENANT_ID = process.env.AZURE_TENANT_ID;
+	if (AZURE_TENANT_ID === undefined) {
+		throw new Error("AZURE_TENANT_ID environment variable is not set");
+	}
+
+	const [photos, setPhotos] = useState<string[]>([]);
+	useEffect(() => {
+        const allPhotos: string[] = [];
+        for (const element of Array.from(onlineUsers.onlineUsers.clientValues())) {
+            for (const user of Array.from(element.items.values())) {
+                allPhotos.push(user.value.value.photo);
+            }
+        }
+        setPhotos(allPhotos);
+    }, [onlineUsers]);
+
+	useEffect(() => {
+		const fetchPhoto = async () => {
+			const credential = new InteractiveBrowserCredential({
+				clientId: AZURE_CLIENT_ID,
+				tenantId: AZURE_TENANT_ID,
+			});
+
+			const authProvider = new TokenCredentialAuthenticationProvider(credential, {
+				scopes: ["User.Read"],
+			});
```

Comment:
Does this mean that the user gets a login prompt when the avatars are going to be rendered?

Reply:
Yes, if the user's microsoft account signed in expired or not exist, there will be a login popup asking for login. I saw that popup in opening loop doc all the time

Reply:
Discussed offline, made code changes as:
1. for sdsp client, user is login so we should already have the credentials
2. tinylicious client -> just show their user id as avatar
3. should always show present users, remove no login not showing online users logic.